code/modules/projectiles/projectile/magic_projectiles.dm

```diff
@@ -324,8 +324,11 @@
 				S.icon = change.icon
 				if(H.mind)
 					H.mind.transfer_to(S)
-					to_chat(S, "<span class='warning'>You are an animated statue. You cannot move when monitored, but are nearly invincible and deadly when unobserved!</span>")
-					to_chat(S, "<span class='userdanger'>Do not harm [firer.name], your creator.</span>")
+					var/list/messages = list()
+					messages.Add("<center><span class='userdanger'>You have been transformed into an animated statue.</span></center>")
+					messages.Add("You cannot move when monitored, but are nearly invincible and deadly when unobserved! Hunt down those who shackle you.")
+					messages.Add("<center>Do not harm [firer.name], your creator.</center>")
```

Comment:
these other lines need the center removed too

Reply:
whoops! I assumed the suggestion before did it. my bad!